app/repositories/QuestionnaireRepository.scala

```diff
@@ -35,7 +35,8 @@ import scala.language.postfixOps
 trait QuestionnaireRepository {
   def addQuestions(applicationId: String, questions: List[PersistedQuestion]): Future[Unit]
   def findQuestions(applicationId: String): Future[Map[String, String]]
-  def onlineTestPassMarkReport: Future[Map[String, PassMarkReportQuestionnaireData]]
+  def findForOnlineTestPassMarkReport: Future[Map[String, QuestionnaireReportItem]]
+  def findAllForDiversityReport: Future[Map[String, QuestionnaireReportItem]]
```

Comment:
This is a subjective point but I don't like names like "findAllForDiversityReport". Surely, the repository is being asked to 'findAll' - what does it care what it's for.